lib/ts/recipe/thirdpartypasswordless/functionOverrides.ts

```diff
@@ -0,0 +1,43 @@
+import { RecipeInterface } from "supertokens-web-js/recipe/thirdpartypasswordless";
+import { RecipeOnHandleEventFunction } from "../recipeModule/types";
+import { OnHandleEventContext } from "./types";
+import { getFunctionOverrides as getThirdpartyFunctionOverrides } from "../thirdparty/functionOverrides";
+import { getFunctionOverrides as getPasswordlessFunctionOverrides } from "../passwordless/functionOverrides";
+import getThirdpartyRecipeImplementation from "./recipeImplementation/thirdPartyImplementation";
+import getPasswordlessRecipeImplementation from "./recipeImplementation/passwordlessImplementation";
+
+export const getFunctionOverrides =
+    (recipeId: string, onHandleEvent: RecipeOnHandleEventFunction<OnHandleEventContext>) =>
```

Comment:
why can;t we directly override all the functions in this recipe impl and then add event handlers for them in here itself?

Reply:
the way it is now we are adding handlers only once without repetition, if i understood your question